jaeger/rhbjaeger-release-notes.adoc

```diff
@@ -0,0 +1,21 @@
+[id="jaeger-release-notes"]
+= Jaeger Release Notes
+include::modules/jaeger-document-attributes.adoc[]
+:context: jaeger-release-notes
+
+toc::[]
+
+// The following include statements pull in the module files that comprise the assembly.
+
+include::modules/jaeger-product-overview.adoc[leveloffset=+1]
+
+include::modules/support.adoc[leveloffset=+1]
```

Comment:
in support.adoc, 
s/please submit a Bugzilla report at http://bugzilla.redhat.com against/please submit a lilnk:http://bugzilla.redhat.com[Bugzilla report] against/

I know it's in a module, but that link shouldn't be naked.